recipe/cudatoolkit-dev-post-install.py

```diff
@@ -99,68 +81,44 @@ def create_activate_and_deactivate_scripts(self):
         activate_scripts_dir = scripts_dir / "activate.d"
         deactivate_scripts_dir = scripts_dir / "deactivate.d"
 
-        activate_scripts_list = [
-            "cudatoolkit-dev-activate.sh",
-            "cudatoolkit-dev-activate.bat",
-        ]
+        activate_scripts_list = ["cudatoolkit-dev-activate.sh"]
         for file_name in activate_scripts_list:
             file_full_path = activate_scripts_dir / file_name
             shutil.copy(file_full_path, activate_dir_path)
 
-        deactivate_scripts_list = [
-            "cudatoolkit-dev-deactivate.sh",
-            "cudatoolkit-dev-deactivate.bat",
-        ]
+        deactivate_scripts_list = ["cudatoolkit-dev-deactivate.sh"]
 
         for file_name in deactivate_scripts_list:
             file_full_path = deactivate_scripts_dir / file_name
             shutil.copy(file_full_path, deactivate_dir_path)
 
+    def download(self, url, target_full_path):
+        cmd = ["wget", url, "-O", target_full_path, "-q"]
+        try:
+            subprocess.check_call(cmd)
+        except subprocess.CalledProcessError as exc:
+            raise exc
+
     def download_blobs(self):
-        """Downloads the binary blobs to the $SRC_DIR
+        """Downloads the binary blobs to the $BLOB_DIR
         """
         dl_url = urlparse.urljoin(self.base_url, self.installers_url_ext)
         dl_url = urlparse.urljoin(dl_url, self.cu_blob)
-        dl_path = os.path.join(self.src_dir, self.cu_blob)
-        if not self.debug_install_path:
-            print("downloading %s to %s" % (dl_url, dl_path))
-            download(dl_url, dl_path)
-
-        else:
-            existing_file = os.path.join(self.debug_install_path, self.cu_blob)
-            print("DEBUG: copying %s to %s" % (existing_file, dl_path))
-            shutil.copy(existing_file, dl_path)
-
-    def check_md5(self):
-        """Checks the md5sums of the downloaded binaries
-        """
-        md5file = self.md5_url.split("/")[-1]
-        path = os.path.join(self.src_dir, md5file)
-        download(self.md5_url, path)
-
-        # compute hash of blob
-        blob_path = os.path.join(self.src_dir, self.cu_blob)
-        md5sum = hashsum_file(blob_path, "md5")
-
-        # get checksums
-        with open(path, "r") as f:
-            checksums = [x.strip().split() for x in f.read().splitlines() if x]
+        dl_path = os.path.join(self.blob_dir, self.cu_blob)
 
-        # check md5 and filename match up
-        check_dict = {x[0]: x[1] for x in checksums}
-        assert check_dict[md5sum].startswith(self.cu_blob[:-7])
+        print("downloading %s to %s" % (dl_url, dl_path))
```

Comment:
I have found `debug_install_path` feature useful to save time in debugging. How about:
```
        local_path = os.path.join(os.path.dirname(os.environ['RECIPE_DIR']),
                                   os.path.basename(dl_path))
        if os.path.isfile(local_path):
            print("copying %s to %s" % (local_path, dl_path))
            shutil.copyfile(local_path, dl_path)
        else:
            print("downloading %s to %s" % (dl_url, dl_path))
            self.download(dl_url, dl_path)
```
that will copy the cuda installer from the parent directory of recipe if it exists?

Reply:
Although this changeset does not apply when conda-build runs the tests - the cuda installer download is still triggered...